### DISCUSSION TITLE
[M-Z] (2024/25) - sezione 0.7 - es. 3.01 (28 ottobre)

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/calcolo-delle-probabilita#302] [Esercizi M-Z]
![image](https://github.com/user-attachments/assets/0ecbb89c-b632-40df-b21e-6ce0ffb2632a)

**Comment by rimaout** [comment]
<img width="843" alt="image" src="https://github.com/user-attachments/assets/ed8d7f5d-a11e-41f1-ae1a-3300b5be6fd1">

**Comment by SickRickX2** [comment]
![image](https://github.com/user-attachments/assets/faa853e8-f24b-46d9-81c9-4dfc370f66f6)

**Comment by marcoseverini** [comment]
![immagine](https://github.com/user-attachments/assets/619df34c-61fa-49f8-a313-0c4d93432708)
